apps/etterlatte-behandling/src/main/kotlin/oppgaveny/OppgaveServiceNy.kt

```diff
@@ -71,6 +76,22 @@ class OppgaveServiceNy(private val oppgaveDaoNy: OppgaveDaoNy, private val sakDa
             )
         )
     }
+    fun opprettNyOppgaveMedSakOgReferanseOgSaksbehandler(
```

Comment:
Denne er ikke i bruk, glemte du å fjerne den eller skulle den bli brukt i stedet for den andre ?

Reply:
lagde den egentlig for neste steg, dvs funksjonalitet for å kunne sende en oppgave tilbake til saksbehandler